build/pxc-entrypoint.sh

```diff
@@ -252,6 +252,10 @@ if [ -z "$CLUSTER_JOIN" ] && [ "$1" = 'mysqld' -a -z "$wantHelp" ]; then
 			ALTER USER 'root'@'localhost' IDENTIFIED BY '${MYSQL_ROOT_PASSWORD}' ;
 			GRANT ALL ON *.* TO 'root'@'localhost' WITH GRANT OPTION ;
 			${rootCreate}
+			REVOKE SYSTEM_USER ON *.* FROM root ;
+
+			CREATE USER 'operatoradmin'@'${MYSQL_ROOT_HOST}' IDENTIFIED BY '${OPERATOR_ADMIN_PASSWORD}' ;
```

Comment:
why not, just `operator`?
```suggestion
			CREATE USER 'operator'@'${MYSQL_ROOT_HOST}' IDENTIFIED BY '${OPERATOR_ADMIN_PASSWORD}' ;
```

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/436/commits/8f0d3d72ea10026c22a82cce1e00528083f56ef7